subscription-manager.spec

```diff
@@ -798,6 +799,22 @@ of Red Hat subscriptions. There are many such interfaces, subscription-manager-g
 subscription-manager-initial-setup-addon, and subscription-manager-cockpit-plugin primarily.
 %endif
 
+
+%package -n python3-cloud-what
+Summary: Python package for detection of public cloud provider
+License: GPLv2
+BuildArch: noarch
+Requires: python3-requests
+%ifnarch aarch64 ppc ppc64 ppc64le s390 s390x
+Requires:  %{py_package_prefix}-dmidecode %{?dmidecode_version}
+%endif
+
+%description -n python3-cloud-what
+This package contains Python module for detection public cloud. The
```

Comment:
This package contains a Python module for detection and collection of public cloud metadata and signatures.

^ My suggestion for the wording of this. Again non-nacking.

Reply:
Good suggestion. Thanks.